Go Client/GameState.cpp

```diff
@@ -63,29 +63,29 @@ stone GameState::WhosMoveJustWent()
 
 uint16_t GameState::PrizonerStones(stone stone)
```

Comment:
inconsistent naming here: could probably stick to either prisoner stones or dead stones instead of both?